### PR TITLE
Add announcement system

### PR DESCRIPTION
**Pull request by morganbarber**
Topbar on 404.html and index.html

The announcement can be changed by modifying `announcement.txt`

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -2,7 +2,42 @@
 <html>
     <head>
         <title>Error 404</title>
-    </head>
+        <style> .announcement-banner {
```
Comment (dumbmonkey2) [inline 404.html]: Fix this styling

File: 404.html [hunk 2]
```diff
@@ -2,7 +2,42 @@
 <html>
     <head>
         <title>Error 404</title>
-    </head>
+        <style> .announcement-banner { 
+            position: fixed; 
+            top: 0; left: 0;
```
Comment (dumbmonkey2) [inline 404.html]: Should be 2 lines

File: 404.html [hunk 3]
```diff
@@ -2,7 +2,42 @@
...
+            background-color: #f8f8f8; 
+            padding: 10px; 
+            text-align: center; 
+            z-index: 9999; 
+        } 
+        .announcement-banner .close-button { 
+            float: right; 
+            cursor: pointer; 
+        } 
+        </style>
```
Comment (dumbmonkey2) [inline 404.html]: Should this go in the style.css file if its being used by 2 files?

File: 404.html [hunk 4]
```diff
@@ -2,7 +2,42 @@
...
+          fetch("announcement.txt").then(function(response) {
+              return response.text();
+          }).then(function(text) {
+              announcementBanner.innerHTML = text;
+              announcementBanner.style.display = "block";
+          }).catch(function(error) {
+              console.error("Error fetching announcement:", error);
+          });
+        }); 
+        </script>
```
Comment (dumbmonkey2) [inline 404.html]: Fix indentation issues please

File: index.html [hunk 5]
```diff
@@ -17,8 +17,42 @@
...
+      fetch("announcement.txt").then(function(response) {
+          return response.text();
+      }).then(function(text) {
+          announcementBanner.innerHTML = text;
+          announcementBanner.style.display = "block";
+      }).catch(function(error) {
+          console.error("Error fetching announcement:", error);
+      });
+    }); 
+    </script>
```
Comment (dumbmonkey2) [inline index.html]: Fix indentation issues

File: index.html [hunk 6]
```diff
@@ -17,8 +17,42 @@
...
+      background-color: #f8f8f8; 
+      padding: 10px; 
+      text-align: center; 
+      z-index: 9999; 
+      } 
+      .announcement-banner .close-button { 
+        float: right; 
+        cursor: pointer; 
+      } 
+    </style>
```
Comment (dumbmonkey2) [inline index.html]: Same with index.html, should this go in style.css since its being used in 2 files?

File: 404.html (hunk 3 above)
Comment (morganbarber) [reply 404.html]: Sure

File: index.html (hunk 6 above)
Comment (morganbarber) [reply index.html]: Sure

File: 404.html (hunk 1 above)
Comment (morganbarber) [reply 404.html]: Fixed when creating new file

File: 404.html (hunk 2 above)
Comment (morganbarber) [reply 404.html]: Fixed when creating new file

**Review by dumbmonkey2 — APPROVED** [review]
Good

**Pull request merged**